inf-clojure.el

```diff
@@ -667,7 +667,7 @@ process buffer for a list of commands.)"
                                           'confirm-after-completion))))
   (if (not (comint-check-proc "*inf-clojure*"))
       ;; run the new process in the project's root when in a project folder
-      (let ((default-directory (clojure-project-dir))
+      (let ((default-directory (or (clojure-project-dir) "./"))
```

Comment:
Why not just to default to `default-directory` in this case?

Reply:
great point! thanks!

Reply:
I see my remark as resolved, but the code seems the same to me.

Reply:
weird. i thought i changed this and force pushed. i'll fix it tonight. sorry!

Reply:
No worries!